test/infrastructure/docker/api/v1beta1/dockermachinetemplate_types.go

```diff
@@ -28,6 +28,7 @@ type DockerMachineTemplateSpec struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:path=dockermachinetemplates,scope=Namespaced,categories=cluster-api
 // +kubebuilder:storageversion
+// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels['machine\\.cluster\\.x-k8s\\.io/cluster-name']",description="Cluster"
```

Comment:
```suggestion
```
We don't set a cluster-name label on this resource

Reply:
Agreed, also with ClusterClasses, templates are going to be shared across many clusters.